indico/modules/designer/templates/backside_list.html

```diff
@@ -0,0 +1,82 @@
+{% extends 'layout/dialog_base.html' %}
+
+{% block description %}
+    {%- trans -%}
+        Listed below you will find the badge templates that can be used as the back side for your template. Note that only single-sided templates with the same dimentions can be used as the back side of a template.
+    {%- endtrans -%}
+{%- endblock %}
+
+{% block content %}
+    {% set template_width = current_template.data.width %}
+    {% set template_height = current_template.data.height %}
+    <section>
+        <h3>{% trans %}Inherited templates{% endtrans %}</h3>
+        {% if inherited_templates %}
+            <div class="i-box just-group-list">
+                <div class="i-box-content">
+                    <ul class="group-list no-content-before badge-backside-list">
+                        {% for tpl in inherited_templates | sort(attribute='title') if tpl.data.width == template_width and tpl.data.height == template_height %}
+                            <li>
+                                <a href="#" class="backside-template"
+                                   data-href="{{ url_for('.get_template_data', tpl) }}">
+                                    <i class="designer-template-type-icon template-icon-{{ tpl.type.name }}"
+                                       title="{{ tpl.type.title }}"></i>
+                                    <span>{{ tpl.title }}</span>
+                                </a>
+                           </li>
+                       {% endfor %}
+                    </ul>
+                </div>
+            </div>
+        {% else %}
+            <div class="italic text-not-important">
+                {% trans %}No templates{% endtrans %}
+            </div>
+        {% endif %}
+    </section>
+    <section class="custom-template-list">
+        <h3>{% trans %}Custom templates{% endtrans %}</h3>
+        {% if custom_templates %}
+            <div class="i-box just-group-list">
+                <div class="i-box-content">
+                    <ul class="group-list no-content-before badge-backside-list">
+                        {% for tpl in custom_templates | sort(attribute='title') if tpl.data.width == template_width and tpl.data.height == template_height %}
```

Comment:
I discussed this with @pferreir on Friday and that's what we concluded.. @pferreir what do you think?